IFC4x3/Properties/m/MechanicalTension_0NjLrbe5v3zuECrs8qsUAO/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="MechanicalTension_0NjLrbe5v3zuECrs8qsUAO" Name="MechanicalTension" UniqueId="17b55d65-a05e-43f7-838c-d76234d9e298" PrimaryDataType="IfcForceMeasure" />
```

Comment:
Similar property to  CatenaryWireMechanicalTension and others, should they be harmonized?

Reply:
I have kept this property (mechanicalTension) and mapped all concept to it (CatenaryWire / StichWire) Commit : 305806220fb93c9010687f137214b3f48df310cb

Reply:
Reuse TensileStrength?